transformer_engine/pytorch/fp8.py

```diff
@@ -515,7 +515,7 @@ def _update_amax_history(amax_history: torch.Tensor) -> torch.Tensor:
     return amax_history
 
 
-@jit_fuser
+# @jit_fuser
```

Comment:
@sudhakarsingh27 why did you remove the fusion of these functions?

Reply:
We can probably restore them. I suspect it's a debugging artifact.